### ISSUE TITLE
Scrollbars drag

### ISSUE THREAD

**Issue by magicalskye** [Slion/Fulguris#497]
Hello, I am really liking this little browser, but it has the same accessibility problem as all the others do. It has no usable scroll bar. I am a disabled person and I use my tablet to read long texts, and I need to be able to actually grab and pull the scroll bar, as scrolling with gestures really, really hurts.

Currently this is not possible.

If you added an accessibility option to make the scrollbar wider, always visible ("visible" as in "with decent contrast"), and actually grabbable, you would achieve the distinction of offering the only open source mobile browser that is actually comfortably usable for people with this type of mobility impairment. (The only mobile browser I've been able to find so far that offers something like this *at all* is Opera, and the way they do it is still borderline useless for someone with impaired mobility, because you need to grab a tiny button that is also almost impossible to see, and it only appears when you are already scrolling.) An alternative would be to make it possible to "pull out" the scroll bar by swiping from the side, but this would also need to be possible at all times, not only when you are already scrolling.

Thank you for your work, wishing a good day to everyone who reads this.

**Comment by Slion** [comment]
Thanks for your feedback. I'll see what I can do.

**Comment by Slion** [comment]
@magicalskye Unfortunatly I have not found a way to achieventhe the scollbar drag in touch mode. 
There might ways to get it done but it could take days to get it to work.
Could using a mouse be a solution for you?

**Comment by magicalskye** [comment]
i only use a mouse on the computer, when i’m in bed (much of the time, because i am very sick) it causes more problems than it solves :/ i will survive without, thank you for trying at least!

**Comment by Slion** [comment]
What you could do is report that issue to the WebView project or maybe even to the AOSP. They should be able to come up with a solution for scrollbars to be draggable in touch mode.

**Comment by Slion** [comment]
I pushed some changes that will allow you to customize the scrollbars but you still won't be able to grab them without a mouse or pointer device.
![image](https://github.com/Slion/Fulguris/assets/6508892/cda4b5bc-188d-4653-9afe-1b38c82018b3)

**Comment by Slion** [comment]
@magicalskye Actually I just tested it on a Samsung Tab S8 Ultra and there you can actually grab the scrollbars. So it may depend of your device. On my Huawei P30 Pro for instance I can't grab them no matter how I tweak the settings.

What device are you using? I'll try do a release soon so that you can test it yourself.

**Comment by Slion** [comment]
It also works on a Samsung Galaxy A22 running Android 13 🥳😁
Not sure why it ain't working on Huawei P30 Pro running Android 10. Maybe it's something they fixed in later Android version or it's something specific the Samsung/Huawei.

**Comment by Slion** [comment]
Here is what it looks like:
![image](https://github.com/Slion/Fulguris/assets/6508892/93a3e98b-920e-454e-b895-0de52fa437cb)

**Comment by Slion** [comment]
It's not working on an LG G8X running Android 12.

**Comment by Slion** [comment]
Otherwise it might be possible to put the WebView inside a ScrollView:
https://stackoverflow.com/questions/23518056/what-are-the-options-for-forcing-a-scrollbar-in-an-android-webview

**Comment by magicalskye** [comment]
I have a Samsung Tab A and i cannot grab any scrollbars (in any app that i’ve tried), but then again i am also in Europe so who knows what the difference is

so basically this is an android configuration issue? that is so messed up, lol. locked mobile OSes were a mistake

**Comment by Slion** [comment]
I'm in Europe too. It's really odd cause even ancient Samsung devices I tested worked fine.
Which model of the Tab A do you have? You can check this out using [Specs](https://slions.net/resources/specs.34/).

Did you configure the scrollbar to remain on screen and with larger size? Can't you grab it then?
You can access the scrollbars settings from Menu > Options > Scrollbars.

**Comment by Slion** [comment]
I did my best to make scrollbars more accessible. The drag itself seems to be device dependent for some weird reason.
